x/concentrated-liquidity/README.md

```diff
@@ -1171,6 +1171,14 @@ to only a few denoms.
 Our list at launch is expected to consist of OSMO, DAI and USDC. These are set in the v16
 upgrade handler.
 
+- `IsPermisionlessPoolCreationEnabled` bool
+
+The flag indicating whether permissionless pool creation is enabled or not. For launch, we have
+decided to disable permissionless pool creation. It will still be enabled via governance.
+This is because we want to limit the number of pools for risk management and want to avoid
+fragmenting liquidity for major denom pairs with configurations of exponent at price one
+and tick spacing that are not ideal.
```

Comment:
The latter point will more than likely be moot since we are selecting a canonical exponent and tick spacing now

Reply:
Tick spacing is still configurable, isn't it?

Reply:
I thought we wanted to do 100 all around? Even if that isn't the case, we should change it to say "want to avoid
fragmenting liquidity for major denom pairs with configurations of ~~exponent at price one
and~~ tick spacing that are not ideal.

Reply:
done